commands/link/__tests__/__fixtures__/explicit-local/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commands": {
+    "link": {
+      "localDependencies": "explicit"
```

Comment:
We're missing CLI option config for `--local-dependencies`, as well as README documentation. It's especially burdensome for an option like this because it is such a _broad_ thing. Perhaps instead of a flag, we need to centralize the "am i using a custom protocol" logic better?
* https://github.com/lerna/lerna/blob/448f2aee7258febc15c131c1128688326a52778f/commands/publish/index.js#L519-L522
* https://github.com/lerna/lerna/blob/448f2aee7258febc15c131c1128688326a52778f/commands/version/index.js#L522-L524
* https://github.com/lerna/lerna/blob/448f2aee7258febc15c131c1128688326a52778f/commands/bootstrap/index.js#L240-L248

After all, in an ideal world, the `workspace:` protocol would "just work", similar to how `file:`/`link:` is handled today.